<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewRecepie</class>
 <widget class="QDialog" name="DialogNewRecepie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>НОВА РЕЦЕПТА</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ИМЕ НА РЕЦЕПТА</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRecepieName">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ЕТИКЕТ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTag">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </row>
     <column>
      <property name="text">
       <string>СЪСТАВКА</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>КОЛИЧЕСТВО</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="10" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="10" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ЗАПАЗИ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ОТКАЖИ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewRecepie</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
